### ISSUE TITLE
[MODULE] - Difficult words extractor

### ISSUE THREAD

**Issue by LeonardPuettmann** [code-kern-ai/bricks#227]
**Please describe the module you would like to add to bricks**
I would like to have a module that is able to extract difficult words from a text. 

**Do you already have an implementation?**
If so, please share it here. For instance:
```python
from typing import Dict, any
def my_module(record: Dict[str, Any]) -> str:
    input_text = record[YOUR_ATTRIBUTE].text
    return textstat.difficult_words(input_text)
```

**Additional context**
This should be possible with the textstat module.

**Comment by rpavani1998** [comment]
Hi there, I've been investigating this issue and I was looking into the textstat module, I understood that difficult words are extracted based on the syllable count, with a default threshold of 2. I'm wondering if we could add a parameter that allows the user to adjust this threshold here. What do you think about this?

**Comment by jhoetter** [comment]
Hi @rpavani1998, You're absolutely right, we could turn this into a parameter that can be adjusted by the user. Either @LeonardPuettmannKern  will look into this sometime soon (he's currently in the last steps of a really cool bricks restructuring, which will add GDPR-badges and use case tags), or you can suggest a PR so we see the change. Generally, the change should be easy in that we put that parameter into the brick itself as a variable 🙂

**Comment by rpavani1998** [comment]
Added with #274

**Comment by LeonardPuettmannKern** [comment]
Implemented in #290  by @rpavani1998 !

**Comment by jhoetter** [comment]
This could be really, really interesting for reference complexity estimation in RAG
